docs/tutorials/gcp_gpu.md

```diff
@@ -175,6 +175,20 @@ After a moment, you should see a result that looks similar to the following.
 [(0.7071067690849304+0j), 0j]
 ```
 
+### Optional: Use the NVIDIA cuQuantum SDK 
+
```

Comment:
Could you add that the environment variable `CUQUANTUM_ROOT` should be set to the path to cuQuantum before running `pip install .`?

Reply:
Added a note about this. Unsure if it would be preferable to split this note so that the step 5 part appears in step 5, or if it's better to keep as a single block - let me know if you have a preference.

Reply:
Looks good to me.